setup.py

```diff
@@ -85,7 +85,7 @@ def run(self):
 
 
 setup(
-    name="lephare",
+    name="lephare-dev",
```

Comment:
Temporary change required by pypi. It requires that the `name` be the same as the repository name.